SR2025/2024-09-02-competition-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+to: Potential team supervisors
+subject: Announcing Student Robotics 2025!
+---
+
+Hi,
+
+We're pleased to [announce][announcement] that registration for Student Robotics
+2025 is now open. If you would like to compete, please complete the
+[signup form][signup-form] with the details of your team. All future emails
+about SR2025 will be sent to the details you provide there.
+
+This year we're hosting [Kickstart][kickstart] at the University of Southampton,
+on Saturday 12th October. We will also be live-streaming the event for those
+unable to attend in person. We expect to confirm places towards the end of
+September.
```

Comment:
Slightly concerned that "end of September" -> 12th October feels quite short -- do we think that this might put people off? Especially since we're only announcing at the start of September.

Reply:
https://github.com/srobo/team-emails/pull/188/commits/bede9ebdebe9339da14e64581ee2a055f83d07b7